docs/content/preview/architecture/docdb-replication/async-replication.md

```diff
@@ -249,7 +249,7 @@ When the source universe is lost, an explicit decision must be made to switch ov
 
 ### Bootstrapping replication
 
-- Currently, it is your responsibility to ensure that a target universe has sufficiently recent updates so that replication can safely resume.  In the future, bootstrapping the target universe will be automated, which is tracked in [#11538](https://github.com/yugabyte/yugabyte-db/issues/11538).
+- Currently, it is your responsibility (see [these instructions](../../../deploy/multi-dc/async-replication/#bootstrap-a-target-universe))   to ensure that a target universe has sufficiently recent updates so   that replication can safely resume.  In the future, bootstrapping the   target universe will be automated, which is tracked in   [#11538](https://github.com/yugabyte/yugabyte-db/issues/11538).
```

Comment:
```suggestion
- Currently, it is your responsibility to ensure that a target universe has sufficiently recent updates so that replication can safely resume (for instructions, refer to [Bootstrap a target universe](../../../deploy/multi-dc/async-replication/#bootstrap-a-target-universe)).  In the future, bootstrapping the target universe will be automated, which is tracked in [#11538](https://github.com/yugabyte/yugabyte-db/issues/11538).
```

Reply:
:+1: